src/components/node-list/index.js

```diff
@@ -87,41 +102,64 @@ const NodeListProvider = ({
   const onItemMouseLeave = (item) => {
     if (isTagType(item.type)) {
       onToggleTagActive(item.id, false);
+    } else if (isModularPipelineType(item.type)) {
+      onToggleModularPipelineActive(item.id, true);
```

Comment:
⚠️

I think this should be `onToggleModularPipelineActive(item.id, false)` - this appears to be the cause of a bug that results in modular pipelines staying active (highlighted) once hovered over, even after mouseleave. Try hovering on/off and see if it behaves as you'd expect!

Reply:
great spot - thank you!